Sound/Tidal/Pattern.hs

```diff
@@ -644,7 +662,7 @@ irand i = (floor . (* (fromIntegral i))) <$> rand
 d1 $ sound (samples "xx(3,8)" (tom $ choose ["a", "e", "g", "c"]))
 @
 
-plays a melody randomly choosing one of the four notes: `"a"`, `"e"`, `"g"`, `"c"` 
+plays a melody randomly choosing one of the four notes: `"a"`, `"e"`, `"g"`, `"c"`
```

Comment:
not sure, but since you started using haddock style comments more strictly, backticks won't get recognized, what do we want here, to have the notes be printed as typewriter-text, signify that they are strings, or more precisely `Pattern String`?